sample-code/src/server.ts

```diff
@@ -35,8 +36,26 @@ app.get('/embedding', async (req, res) => {
   }
 });
 
-app.get('/orchestration', (req, res) => {
-  res.status(418).send('Not implemented 🛠️');
+app.get('/orchestration', async (req, res) => {
+  try {
+    res.send(await orchestrationCompletion());
+  } catch (error: any) {
+    console.error(error);
+    res
+      .status(500)
+      .send('Yikes, vibes are off apparently 😬 -> ' + error.message);
+  }
+});
+
+app.get('/ai-core/get-deployments', async (req, res) => {
+  try {
+    res.send(await chatCompletion());
```

Comment:
Is this correct? shouldn't we call getDeployments function?

Reply:
oopsie, good catch